### PR TITLE
Add support for UpdateOptions when updating email/phone

### PR DESCRIPTION
**Pull request by shilgapira**
## Description
- Add support for `UpdateOptions` when updating email/phone

## Must
- [X] Tests
- [X] Documentation (if applicable)

### CODE REVIEW

File: lib/src/internal/http/descope_client.dart [hunk 1]
```diff
@@ -41,18 +41,22 @@ class DescopeClient extends HttpClient {
     });
   }
 
-  Future<MaskedAddressServerResponse> otpUpdateEmail(String email, String loginId, String refreshJwt) {
+  Future<MaskedAddressServerResponse> otpUpdateEmail(String email, String loginId, String refreshJwt, UpdateOptions? options) {
     return post('auth/otp/update/email', MaskedAddressServerResponse.decoder, headers: authorization(refreshJwt), body: {
       'loginId': loginId,
       'email': email,
+      "addToLoginIDs": options?.addToLoginIds,
```
Comment (asafshen) [inline lib/src/internal/http/descope_client.dart:48]: Mind the double quotes (vs single above)

**Review by asafshen — APPROVED** [review]
Cool!

File: lib/src/internal/http/descope_client.dart (hunk 1 above)
Comment (shilgapira) [reply lib/src/internal/http/descope_client.dart:48]: Good catch

Comment (shilgapira) [reply lib/src/internal/http/descope_client.dart]: Done

**Review by asafshen — APPROVED** [review]
🙌

**Pull request merged**